tablo/models.py

```diff
@@ -771,10 +801,17 @@ def update_feature(self, feature):
                 )
                 c.execute(set_geom_command, [primary_key])
 
+        # save out large images
+        for key, value in images_large.items():
+            image_path = key.replace("NO_PRIMARY_KEY", str(primary_key))
+            FeatureServiceLayer.save_image(value, image_path, LARGE_IMAGE_NAME)
+
         return primary_key
 
     def delete_feature(self, primary_key):
 
+        from django.core.files.storage import default_storage as image_storage
```

Comment:
The reason for including the import nested in a function is to keep it from being executed when the file itself is imported. Generally, this is to get around circular dependencies (X imports Y which import X). It can also be used for conditional imports (only importing when using a certain feature). In general, though, if there's not a reason for importing it in a special way, it's best just to keep it at the top of the file.